contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -267,6 +267,20 @@ var BaseWorkspaceView = BaseView.extend({
 		// Recalculate counts
 		this.reload_ancestors(original_parents, true);
 	},
+	sync_content:function(){
+		var SyncView = require("edit_channel/sync/views");
+		$("#main-content-area").append("<div id='dialog'></div>");
+		var sync = new SyncView.TempSyncModalView({
+			el: $("#dialog"),
+		    onsync: this.reload_ancestors,
+		    model: window.current_channel.get_root("main_tree")
+		});
+		// var sync = new SyncView.SyncModalView({
```

Comment:
Mostly keeping this in as this has node diffs. Once I can improve the query time, I would like to move back to this modal